src/Sylius/Behat/Element/Admin/Taxon/FormElementInterface.php

```diff
@@ -45,4 +45,6 @@ public function enable(): void;
     public function disable(): void;
 
     public function isEnabled(): bool;
+
+    public function searchParentTaxonAutocomplete(string $searchTerm): array;
```

Comment:
No need to complex the naming
```suggestion
    public function searchParentTaxon(string $searchTerm): array;
```